src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -66,11 +67,11 @@ def _process_model_before_weight_loading(self, model, **kwargs):
         apply_quantization_config(model, ct_quantization_config, run_compressed=True)
 
     def _process_model_after_weight_loading(self, model, **kwargs):
-        pass
+        model.is_compressed_tensors_quantized = True
 
     @property
-    def is_trainable(self):
-        return False
+    def is_trainable(self) -> bool:
+        return True
```

Comment:
As said earlier, we should probably create another property `is_qat_trainable` if this is what you are doing. This property was used only for peft training. We might need to rebrand it as `is_peft_trainable`.